sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py

```diff
@@ -10,22 +10,45 @@
 from sr_hand_commander import SrHandCommander
 from sr_robot_commander import SrRobotCommander
 from sr_utilities.hand_finder import HandFinder
+from controller_manager_msgs.srv import ListControllers
+from control_msgs.msg import JointTrajectoryControllerState
+from threading import Lock
 
 
 class SrStateSaverUnsafe(object):
-    def __init__(self, name, hand_or_arm="both", hand_h=False):
+    def __init__(self, name, hand_or_arm="both", hand_h=False, save_target=False):
 
         self.__save = rospy.ServiceProxy(
             'save_robot_state', SaveState)
 
         self.__name = name
-
         self.__hand_h = hand_h
-
-        if hand_or_arm == "arm":
-            self.__commander = SrArmCommander()
-
-        elif hand_or_arm == 'hand':
+        self.__save_target = save_target
```

Comment:
Yes, indeed that would be a separate PR. And I mean it mainly for new files.

About it being more correct. that's far from true.